nav2_controller/src/nav2_controller.cpp

```diff
@@ -31,10 +31,14 @@ ControllerServer::ControllerServer()
 : LifecycleNode("controller_server", "", true),
   lp_loader_("nav2_core", "nav2_core::LocalPlanner")
 {
-  RCLCPP_INFO(get_logger(), "Creating controller");
+  RCLCPP_INFO(get_logger(), "Creating controller server");
 
   declare_parameter("controller_frequency", 20.0);
-  declare_parameter("local_controller_plugin", "dwb_core::DWBLocalPlanner");
+  std::vector<std::string> default_name, default_type;
+  default_type.push_back("dwb_core::DWBLocalPlanner");
```

Comment:
I got a bit confused by the names used here, why not instead match a `default_motion_type` (e.g. FollowPath) to a `default_controller_name` (e.g. DWBLocalPlanner)?

Reply:
I think you read this the opposite way I do. When I use `type` I'm referring to the type of plugin and the name is the name of the plugin. 

With that said, I'm open to naming changes for both of these but if there's confusion, I dont think 'type` or `name` should appear at all so there isn't that opportunity. 

Maybe `default_plugin_name` and `default_plugin_type` to be clear its plugin-centric? Open for thoughts